iceoryx_dds/source/iceoryx2dds_app/main.cpp

```diff
@@ -41,7 +41,8 @@ class ShutdownManager
     static iox::posix::Semaphore s_semaphore;
     ShutdownManager() = default;
 };
-iox::posix::Semaphore ShutdownManager::s_semaphore = iox::posix::Semaphore::create(iox::posix::CreateUnnamedSingleProcessSemaphore, 0u).value();
+iox::posix::Semaphore ShutdownManager::s_semaphore =
+    iox::posix::Semaphore::create(iox::posix::CreateUnnamedSingleProcessSemaphore, 0u).value();
```

Comment:
Error handling is missing here. But it's fine for quality level 4

Reply:
Added to #376